specs/CBOR_SETS.md

```diff
@@ -0,0 +1,29 @@
+# Sets for CBOR
+
+This document specifies a tag for sets in Concise Binary Object Representation (CBOR) [1].
+
+    Tag: 258
+    Data item: array
+    Semantics: Set
+    Point of contact: Input Output HK <info@iohk.io>
```

Comment:
Who is the person to enquire for this piece of information? (Sorry for this "spoonfeeding" request, but I'm still relatively new to the company structure)